packages/admin/src/FilamentManager.php

```diff
@@ -168,6 +175,21 @@ public function getGlobalSearchProvider(): GlobalSearchProvider
         return app($this->globalSearchProvider);
     }
 
+    public function renderHook(string $name): ?string
+    {
+        if (! array_key_exists($name, $this->renderHooks)) {
+            return null;
+        }
+
+        $output = '';
+
+        foreach ($this->renderHooks[$name] as $renderHook) {
+            $output .= (string) app()->call($renderHook);
+        }
+
+        return $output;
```

Comment:
We should return an `HtmlString()` object instead so you don't need to escape the output in Blade

Reply:
Yeah, good shout.